### PR TITLE
Feat: add popoverProps prop to <ColorInput/>

### PR DESCRIPTION
**Pull request by kranux**
### 🔦 Summary

We want to have an ability to configure popover from `<ColorInput/>`. 

To achieve that we will pass prop `popoverProps ={{}}` to <ColorInput/>, for example:

```jsx

<ColorInput
  popoverProps={{
    zIndex: 2019
  }}
>
```
### ✅ Checklist
- [ x] 👨‍💻 API change is approved by the UI Infra Developers @mykas 

- 📚 Change is documented
  - [ x] API description

### CODE REVIEW

**Comment by kranux** [comment]
@mykas we already have some `popover` prefixed props in `<ColorInput />` but I assume that it's not in the scope of current PR?

**Comment by mykas** [comment]
We will need to leave those and later deprecated. The deprecation part I will handle later.

**Comment by mykas** [comment]
@shlomitc Please review this part. This is the first component that will use common props pattern for passing popover props. 

**Pros**
- This pattern is great if only one popover or tooltip

**Cons**
- Not so great if multiple components popovers or tooltips inside.

I suggest using different pattern for components that are structured out of multiple system components: `use composition and expose render slots`.

File: src/ColorInput/components/ColorViewer.js [hunk 1]
```diff
@@ -27,9 +27,11 @@ export class ColorViewer extends React.Component {
       onAddColor,
       addTooltipContent,
       placeholder,
+      popoverProps,
     } = this.props;
     return (
       <Popover
+        {...(popoverProps || {})}
```
Comment (mykas) [inline src/ColorInput/components/ColorViewer.js]: I don't think you need this `|| {}` if you have defined the default props value. Just do `{...popoverProps}`.

Comment (kranux) [reply src/ColorInput/components/ColorViewer.js]: Added defaultProps;

**Comment by mykas** [comment]
Looks good. Lets merge it and release it so you could play with it.

**Pull request merged**